src/shared-components/optionButton/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Container from '../container';
+import { FlexContainer, Text, IconWrapper, CheckmarkWrapper } from './style';
+import CheckmarkIcon from '../../svgs/icons/checkmark-icon.svg';
+
+const OptionButton = ({
+  icon,
+  text,
+  onClick,
+  optionType,
+  selected,
+  styleType,
```

Comment:
we probably want to get `...rest` and pass it along to one of these components that makes sense. Maybe the `Container`. One use-case is if we want to pass along any `data-*` attribute. We use that often in our integration tests

Reply:
done